2019-09-05-group.bib

```diff
@@ -34,4 +34,13 @@ @techreport{eldred_dakota_2010
 	year = {2010},
 	doi = {10.2172/991842},
 	file = {Eldred et al. - 2010 - DAKOTA  a multilevel parallel object-oriented fra.pdf:/Users/gwenchee/Zotero/storage/Q5XJE57U/Eldred et al. - 2010 - DAKOTA  a multilevel parallel object-oriented fra.pdf:application/pdf}
-}
```

Comment:
What is this? Make sure you didn't accidentally delete something important.

Reply:
That is the end bracket that holds the entire citation.  If you look, they are all formatted to have that bracket in its own line.

Reply:
Sorry - I should clarify that I checked in the file, and that bracket is still there.  I think it shows thi deletion, and then, immediately below it, it shows it as an addition.